STMicroelectronics/STM32L4_L4+/app/nx_client.c

```diff
@@ -37,6 +39,40 @@ static TX_EVENT_FLAGS_GROUP azure_iot_flags;
 
 static int32_t telemetry_interval = 10;
 
+az_result dcf_ip_gateway_client_entry(
```

Comment:
>dcf_ip_gateway [](start = 10, length = 14)

We need to decide where this gateway will live. I think it should be part of the ulib, something like az_ulib_gateway_ip